2025 Haunted House/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -20,10 +21,14 @@ public class RobotContainer {
   // PCM1 is floor 1 of haunted house, PCM2 is floor 2
   private final PneumaticsControlModule pcm1 = new PneumaticsControlModule(0);
   private final PneumaticsControlModule pcm2 = new PneumaticsControlModule(1);
+  
+  private final SolenoidMech erinMech; 
 
   /** The container for the robot. Contains subsystems, OI devices, and commands. */
   public RobotContainer() {
     // Configure the trigger bindings
+    erinMech = new SolenoidMech(pcm1.makeSolenoid(0), 1, 3, 1);
```

Comment:
Isn't solenoid 0 used by the other mechs?

Reply:
ou yeah we might have remixed the port numbers since this commit, check the [spreasheet](https://docs.google.com/spreadsheets/d/1Jhs5ze12oZ4oZ7q-0G464bEtU1weu__I-QAnYWbvUWY/edit?gid=0#gid=0)
for available ports @eerinn

Reply:
Some of the other PRs have created constants for holding their port number. This is a good way to do it because then you can essentially write that spreadsheet into the code. I do like that you are using something formal to capture the development state of all the features/branches!